tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = packaging, pep8, py27-ldap{0,1,2}
+envlist = packaging, pep8, py35-ldap{0,1,2}
```

Comment:
It looks like setup.py says only `ldap>=2.8` -- is it still worth testing these old versions?

Also, would probably be good to run `py36`, `py37` (and maybe `py38`)?